src/reuse/lint.py

```diff
@@ -257,6 +264,64 @@ def custom_serializer(obj: Any) -> Any:
     )
 
 
+def format_lines(report: ProjectReport) -> str:
+    """Formats data dictionary as plaintext strings to be printed to sys.stdout
+    Sorting of output is not guaranteed.
+    Symbolic links can result in multiple entries per file.
+
+    Args:
+        report: ProjectReport data
+
+    Returns:
+        String (in plaintext) that can be output to sys.stdout
+    """
+    output = StringIO()
+
+    def license_path(lic: str) -> Optional[Path]:
+        "Resolve a license identifier to a license path."
+        return report.licenses.get(lic)
+
+    if not report.is_compliant:
+        # Bad licenses
+        for lic, files in sorted(report.bad_licenses.items()):
+            for file in sorted(files):
+                output.write(_(f"{file}: bad license {lic}\n"))
```

Comment:
Here and elsewhere, `_(f"{value}")` does not correctly translate. The formatting happens before the translation step here, while it should happen after.